src/main/java/it/pagopa/pm/gateway/client/postepay/PostePayClient.java

```diff
@@ -0,0 +1,99 @@
+package it.pagopa.pm.gateway.client.postepay;
+
+import it.pagopa.pm.gateway.dto.microsoft.azure.login.MicrosoftAzureLoginRequest;
+import it.pagopa.pm.gateway.dto.microsoft.azure.login.MicrosoftAzureLoginResponse;
+import it.pagopa.pm.gateway.dto.microsoft.azure.login.UrlEncoded;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.web.client.RestTemplate;
+import org.openapitools.client.model.ResponseURLs;
+import org.openapitools.client.model.PaymentChannel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+public class PostePayClient {
+
+    private final String microsoftAzureLoginMode = "x-www-form-urlencoded";
+
+
+    @Value("${azureAuth.client.postepay.url}")
+    public String MICROSOFT_AZURE_LOGIN_URL_POSTEPAY;
+
+    @Value("${postePay.client.microsoft.azure.login.client.id:1d1828e6-cf67-4ced-a256-7f076cf2750c}")
```

Comment:
Sanity check on the fallback value:
 * Does it make sense for the server not to report an error if this value is missing?
 * Is it correct that the value is public?